src/clients/index.ts

```diff
@@ -27,8 +27,13 @@ const context: IOContext = {
   platform: '',
 }
 
+const regionHeader = context.region !== Region.Production ? { 'x-vtex-upstream-target': context.region } : null
```

Comment:
So can this region come from `vtex config set/get` too? 😻

Reply:
I'll take a look